easybuild/easyblocks/g/gcc.py

```diff
@@ -434,7 +442,7 @@ def build_step(self):
 
                         # ensure generic build when 'generic' is set to True or when --optarch=GENERIC is used
                         # non-generic build can be enforced with generic=False if --optarch=GENERIC is used
-                        if build_option('optarch') == OPTARCH_GENERIC and self.cfg['generic'] != False:
+                        if build_option('optarch') == OPTARCH_GENERIC and self.cfg['generic'] is not False:
```

Comment:
is not False....
why not just just "self.cfg['generic']"
or at least "is True"

Reply:
Ahh, "use False to enforce non-generic regardless".... didn't notice that one.

Reply:
Indeed, it's easy to overlook that one, `False` is a special value here